apps/web/src/server/applications/__tests__/__snapshots__/applications.test.js.snap

```diff
@@ -85,7 +85,7 @@ exports[`applications GET /case-admin-officer should render the open application
                                 </thead>
```

Comment:
looks like the same snapshot update should be done elsewhere as tests are failing. 
the usual -- -u should work